Content.Shared/Access/Components/IdCardConsoleComponent.cs

```diff
@@ -10,7 +10,7 @@ namespace Content.Shared.Access.Components;
 [Access(typeof(SharedIdCardConsoleSystem))]
 public sealed partial class IdCardConsoleComponent : Component
 {
-    public const int MaxFullNameLength = 30;
+    public const int MaxFullNameLength = 32; # SS220 Name-lenght-fix
```

Comment:
Адич ошибся, в c# коментарии пишутся через `//`
```suggestion
    public const int MaxFullNameLength = 32; // SS220 Name-lenght-fix
```